### DISCUSSION TITLE
Menyelesaikan kuis NN dengan kaggle notebook

### DISCUSSION THREAD

**Discussion by DiekiRianMustapa** [teamalgoritma/community#309] [Q&A] [answered]
Kenapa kaggle netbook untuk run data selalu restart secara automatis ? hal ini membuat saya jadi tidak bisa menyelesaikan kuisnya karena installasi library keras di R selalu gagal. tensor flow not found

![image](https://user-images.githubusercontent.com/107328516/190903299-96aa7203-da01-494d-acd6-188bbac30a69.png)

**Comment by divakartika** [comment, marked as answer]
Dear Bapak @DiekiRianMustapa ,

Restart secara otomatis ini diakibatkan karena notebook kekurangan RAM. Dapat dilihat terdapat pita merah di bagian atas yang bertuliskan "Your notebook tried to allocate more memory than is available. It has restarted."

Solusi dari permasalahan ini adalah dengan _log in_ terlebih dahulu sebelum melakukan _copy and edit code_. Dengan _log in_ ke akun Kaggle, Bapak mendapatkan RAM sebesar 16 GB, sementara tanpa _log in_, RAM yang disediakan hanya sebesar 1 GB.

Semoga dapat membantu. Apabila jawaban ini membantu Anda, bisa berikan "Mark as answer" pada comment ini.
